### DISCUSSION TITLE
Homebridge iopool plugin

### DISCUSSION THREAD

**Discussion by GurYN** [iopool/community#15] [Community development]
Hi everyone,

I just made an iopool plugin for Homebridge service. Feel free to test it and give feedback :)

https://github.com/GurYN/homebridge-iopool

**Comment by RRoyceus** [comment]
I have the plugin installed. It shows The PH and ORP's , but I can not see the values
![image](https://github.com/iopool/community/assets/33940987/6f707993-6d4e-4dfb-926f-b34fb163b4f5)

I also have Homebridge installed on my IPhone and Ipads, I can see a ttest switch install on Homebridge and can use it, but I can not see the PH or ORP

**Reply by RRoyceus** [reply]
I should have been clearer.  The can click on the gear and see the values.  The temperature shows up in the Apple Home APP, but not the PH & ORP
